protocol/packages/currencies/Cargo.toml

```diff
@@ -8,7 +8,8 @@ license.workspace = true
 [package.metadata.cargo-each]
 combinations = [
     { tags = ["ci", "$net", "$protocol"], include-rest = true, generics = { "$net" = ["net_dev", "net_test", "net_main"], "$protocol" = ["neutron-astroport-usdc_axelar", "osmosis-osmosis-usdc_axelar", "osmosis-osmosis-usdc_noble"] } },
-    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], include-rest = true }
+    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], include-rest = true },
+    { tags = ["ci", "net_main", "osmosis-osmosis-statom"], include-rest = true }
```

Comment:
```suggestion
    { tags = ["ci", "net_main", "osmosis-osmosis-statom"], include-rest = true },
```

Reply:
I doubt we need this trailing comma since there is no Cargo.toml doing that so far,
@KirilMihaylov do you insist on it? if yes, we have to change that everywhere

Reply:
appended a comma everywhere